### ISSUE TITLE
Incorrect rendering of export modal content

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2010]
## Bug Report

**Problematic behavior**
When trying to read the content of the export modal, it is vocalized as "Modal content to export the document". The user wants to access the content, which they cannot do.

Especially since an `aria-label` attribute on content that is not focusable is not relevant.

**Possible Solution**
Remove the `aria-label` attribute that is present on the `div`

```
<div aria-label="Contenu modal pour exporter le document" class="sc-5b83f41b-0 iTRszN --docs--modal-export-content"><p class="sc-5b83f41b-0 sc-458c56ea-0 izpgQx hVsyQq">Exportez votre document au format .docx, .odt, .pdf ou .html(zip) pour l'imprimer ou le télécharger.</p></div>
```

**Additional context/Screenshots**

<img width="679" height="360" alt="Image" src="https://github.com/user-attachments/assets/a009d808-3e47-4c97-b4db-25a3ab57e7af" />

**Critères RGAA**

Critère 7.1 : [[Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 

There is no negative impact but the modal content is lost to the user.

**Comment by Ovgodd** [comment]
Hello @cyberbaloo ,
The PR related to this issue will include a fix like this for all modals.
